README.md

```diff
@@ -301,10 +306,10 @@ You'll see a view that looks like this:
 * "job name" - each job has a name, like "create_watch"
 * "job type" - backend/private nomenclature for the handler+options for processing a job
 * "queue" - each job creates a queue with the same name as the job. only jobs named X go into the X queue.
-* "retry jobs" - If a job fails and needs to be retried, it will be put on this queue.
-* "scheduled jobs" - Jobs enqueued to be run in th future will be put on a scheduled job queue.
-* "dead jobs" - If a job exceeds its MaxFails count, it will be put on the dead job queue.
-
+* "retry jobs" - if a job fails and needs to be retried, it will be put on this queue.
+* "scheduled jobs" - jobs enqueued to be run in th future will be put on a scheduled job queue.
+* "dead jobs" - if a job exceeds its MaxFails count, it will be put on the dead job queue.
+* "paused workers" - if workers are paused, then no jobs will be processed by any workers in that worker pool until they are unpaused.
```

Comment:
This also needs clarification. It is a paused queue, not a paused worker. The workers will continue processing jobs from other queues.